_posts/2018-11-15-Rust-survey-2018.md

```diff
@@ -0,0 +1,168 @@
+
+We’re seeing similar numbers in users of the current stable release since last year. Perhaps surprisingly, we’re continuing to see a rise in the number of users who use the Nightly compiler in their workflow. For the second year in a row, Nightly usage has continued to rise, and is now over 56% (up from 51.6% of last year).
+
+![Has upgrading the compiler broken your code](/images/2018-11-RustSurvey/11-Has_upgrading_compiler_broken_you.png)
+
+The percentage of people who see a breakage during a routine compiler update has stayed the same since last year, with 7.4% saying they’ve experienced breakage.
+
+![If so how much work to fix it](/images/2018-11-RustSurvey/12-If_so_how_much_work_to_fix.png)
+
+Breakages generally leaned to requiring minor fixes, though some reported having moderate or major fixes to upgrade to the next stable compiler.
+
+![Preferred install method](/images/2018-11-RustSurvey/13-Preferred_install_method.png)
+
+We again see a strong showing for `rustup`, which continues to hold at 90% of Rust installs. Linux distros follow as a distant second at 17%.
+
+![Experience with Rust tools](/images/2018-11-RustSurvey/14-Tool-experience.png)
+
+Tools like `rustfmt` and `rustdoc` had a strong show, with lots of positive support.  Following these is the `clippy` tool -- despite having fewer users, its users enjoy the tool.  The IDE support tools `Rust Language Server` and `racer` had positive support but unfortunately, of the tools surveyed, generated a few more dislike votes and comments. The `bindgen` tool has relatively small userbase.
+
+## **Rust workflow**
+
+![Which platform are you developing on](/images/2018-11-RustSurvey/15-Platform_developing_on.png)
+
+Linux continues to be a powerhouse among Rust developers, holding on to roughly 80% of Rust developers.  Windows usage has grown slightly from 31% last year to 34% this year, its second year in a row of growth.
+
+![Which platforms are you developing for](/images/2018-11-RustSurvey/16-Platforms_targeting.png)
+
+Linux and Windows continued to show strongly as targets for Rust applications. Other platforms held largely the same as last year, with one exception: WebAssembly. The new technology has showed impressive growth, nearly doubling from last year's 13% to this year's 24%.
+
+![What editors do you use](/images/2018-11-RustSurvey/17-Editors.png)
+
+Vim, the front-runner in editors for two years has now finally been bested by VSCode, which grew from 33.8% of Rust developers to 44.4% this year.
+
+## **Rust at work**
+
+![Do you use Rust at work](/images/2018-11-RustSurvey/18-Rust_at_work.png)
+
+Rust continues is slow-and-steady growth in the workplace. We're now seeing year-of-year growth of full-time and part-time Rust, growing from last years 4.4% full-time and 16.6% part-time to this year's **8.9% full-time** and **21.2% part-time**, a doubling of full-time Rust commercial use.  In total, Rust commercial use grew from 21% to just over 30% of Rust users.
```

Comment:
I think you mean "year-over-year" rather than "year-of-year"?

Also "growing from last years 4.4% full-time" should have a possessive "year's"

Reply:
Fixed